script/.build-deb/control

```diff
@@ -0,0 +1,15 @@
+Source: runc
+Maintainer: Open Containers <dev@opencontainers.org>
```

Comment:
Should this actually be the `Open Containers Initiative`, since it's the group as a whole?